lib/identity_pki/token.ex

```diff
@@ -0,0 +1,53 @@
+defmodule IdentityPki.Token do
+  def box(data, opts \\ []) do
+    expires_at = Keyword.get(opts, :expires_at)
+    key = Keyword.get_lazy(opts, :key, &secret_key/0)
+
+    data =
+      if expires_at do
+        Map.put(data, :expires_at, expires_at)
+      else
+        data
+      end
+
+    json =
+      Map.put(data, :r1, random_bytes(8))
```

Comment:
Ruby PKI takes advantage of Ruby's hash keys being encoded in order and makes sure that `r1` is the first key and `r2` is the last to surround the rest of the data as entropy. Elixir maps don't have a deterministic key order so to more closely match that we could do `Jason.encode!([r1, data, r2])`, but we weren't sure if that was sufficient.